tools/sigclient/cmd/sigclient.go

```diff
@@ -326,6 +330,7 @@ func init() {
 	metricsBenchCmd.PersistentFlags().IntP("numQueryIterations", "n", 1, "Number of query iterations loop")
 
 	alertsLoadTestCmd.PersistentFlags().Uint64P("numAlerts", "n", 1, "Number of alerts to create")
+	alertsLoadTestCmd.PersistentFlags().Int8P("runVector", "v", 0, "Run vector: -1 - Explicitly Disable running of Vector, 0 - Optional State: Will try to run vector, but will not stop the test if encountered and issue with Vector, 1 - Explictly Enable running of Vector")
```

Comment:
Is there a reason we want to allow `runVector=0`?

Reply:
Hmm no particular reason, but by default it will try to start the vector server to ingest these logs into SigLens. If it cannot, it won't stop the server and exit. 
But when enabled explicitly, it will stop the server and exit when vector throws an error.